cfme/utils/log_validator.py

```diff
@@ -81,3 +83,14 @@ def _verify_match_logs(self):
         for pattern in self.matched_patterns:
             if pattern not in self.matches:
                 pytest.fail('Expected pattern {} did not match'.format(pattern))
+
+    def wait_for_log_validation(self, **kwargs):
```

Comment:
Cannot one just use

`wait_for(log_validator.validate_logs, handle_exception=True)` instead of using the `wait_for_log_validation`?

Reply:
@jaryn That's what I thought at first too, however, if you look at `_verify_match_logs()`, it raises `pytest.fail` if there are no matches, so we must catch that exception and have our function raise `True` or `False` in order to use wait for.

Reply:
@john-dupuy I see. Maybe worth adding some comments to your function about it?